commands/rank.js

```diff
@@ -0,0 +1,131 @@
+const Discord = require('discord.js');
+const Enmap = require("enmap");
+
+const ALLOWED_RANKS = [
```

Comment:
Might be better defined as an enmap.  Would then be able to set aliases(ex: "plat3"), ordering(for sorting), and make this command usable for other games.